app/services/flowcommerce_spree/webhooks/capture_upserted_v2.rb

```diff
@@ -37,7 +37,7 @@ def process
       def store_payment_capture(order, capture)
         upsert_order_captures(order, capture)
         payments = order.flow_io_payments
-        map_payment_captures_to_spree(order, payments) if payments.present?
+        map_payment_captures_to_spree(order, payments)
```

Comment:
Hi @abalpablo does it make sense to remove this conditional? will not payments always be present now that you're adding a placeholder payment for Klarna? Maybe I'm missing something but it seems that we might be missing out on a useful validation

Reply:
mmmmm that's a good point, i actually removed it because as you said, we will always have payments. Looks like i gave it one to many twists of mind, if we will have payments we should check for them :) 

Nice catch! I'll add it back